vmware/ngc/NGC-Plugin/html/src/main/webapp/assets/css/storagepool.css

```diff
@@ -0,0 +1,48 @@
+<!-- CSS goes in the document HEAD or added to your external stylesheet -->
+<style type="text/css">
```

Comment:
Is this a demo file? The instructions suggest that the style tag should go into the HEAD and table tag goes into the document BODY. Please check if this file is needed and what should be the content.

Reply:
Thank you for your review. We will fix this problem here.

Reply:
Done